6.x/getting-started-videos.md

```diff
@@ -23,129 +23,137 @@ Mentioned in this video:
 <a name="video-2"></a>
 <h2 class="mb-7">2. Installation & Setup</h2>
 
+Let's setup a laravel project and follow the Backpack installation to get started.
+
 <div class='mx-n1 mt-7 embed-container'><iframe src='https://player.vimeo.com/video/840026719' frameborder='0' webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe></div>
 
 <br>
-Mentioned in this video: // TODO
-- xx
-- yy
+Mentioned in this video:
+- [Laravel docs - create project](https://laravel.com/docs/master/installation#your-first-laravel-project)
+- [Backpack docs - installation](/docs/{{version}}/installation)
 
 
 ----
 
 <a name="video-3"></a>
 <h2 class="mb-7">3. Look & Feel - Introduction to Backpack Themes</h2>
-
+Watch! how to customize the look & feel of admin panel, setup a bootstrap theme of your choice.
```

Comment:
A quick intro to Backpack themes - how to customize the look & feel of your admin panel, or use a Bootstrap theme of your choice.